db/migrations/versions/initial_mirgration.py

```diff
@@ -0,0 +1,53 @@
+"""empty message
+
+Revision ID: initial_migration
+Revises:
+Create Date: 2022-07-15 12:23:02.0
+
+"""
+import sqlalchemy as sa
+import sqlalchemy_utils
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "initial_migration"
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "Code",
+        sa.Column("id", sa.String(length=20), nullable=False),
+        sa.Column("description", sa.Text(), nullable=True),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_table(
+        "Event",
+        sa.Column(
+            "id",
+            sqlalchemy_utils.types.uuid.UUIDType(binary=False),
+            nullable=False,
+        ),
+        sa.Column("code", sa.String(length=20), nullable=False),
```

Comment:
Might be overkill here as this will mean every event will need to created as a code before it can be used as an event? I guess it might be easier to leave this as free text for now. Or have the body as JSON to allow for a free-er structure?

Reply:
Happy to try this approach for now, and we can re-think if it gets too difficult to maintain

Reply:
okay I think that's a fair point though - maybe like you say we trial this strict solution and we can amend if required